cvmfs/cloud_testing/run.sh

```diff
@@ -17,35 +18,12 @@ script_location=$(cd "$(dirname "$0")"; pwd)
 . ${script_location}/common.sh
 
 config_package_base_url="https://ecsft.cern.ch/dist/cvmfs/cvmfs-config"
-
-# static information (check also remote_setup.sh and remote_run.sh)
-cvmfs_workspace="/tmp/cvmfs-test-workspace"
-cvmfs_source_directory="${cvmfs_workspace}/cvmfs-source"
-cvmfs_log_directory="${cvmfs_workspace}/logs"
-
-# global variables for external script parameters
-testee_url=""
-client_testee_url=""
-platform=""
-platform_run_script=""
-platform_setup_script=""
-ec2_config=""
-ami_name=""
-log_destination="."
-username="root"
-userdata=""
-
-# package download locations
-server_package=""
-client_package=""
-devel_package=""
-config_packages=""  # might be more than one... BEWARE OF SPACES!!
-unittest_package=""
-source_tarball="source.tar.gz" # will be prepended by ${testee_url} later
```

Comment:
all those variables come through the environment now, aren't they? To make this clear, I'd like to have them all in the `CT_CAPITALS_...` scheme.